tests/rpc/test_rpc_apiserver.py

```diff
@@ -698,8 +699,8 @@ def test_api_edge_disabled(botclient, mocker, ticker, fee, markets):
     assert rc.json() == {"error": "Error querying /api/v1/edge: Edge is not enabled."}
 
 
-# TODO-lev: @pytest.mark.parametrize('is_short', [True, False])
-def test_api_profit(botclient, mocker, ticker, fee, markets):
+@pytest.mark.parametrize('is_short', [True, False])
+def test_api_profit(botclient, mocker, ticker, fee, markets, is_short):
```

Comment:
Thinking about this - i'm not 100% sure about this way to test this part of the code - as it always only tests one side (either 100% long trades, or 100% short trades).

potentially / ideally, we should have a way to create "mixed trades" - where some are long, and some are short trades.

@samgermain, what do you think about that? (i think i can add that pretty easily in to `create_mock_trades()` if you feel that would make sense).